sources/views/data.js

```diff
@@ -0,0 +1,32 @@
+import { JetView } from "webix-jet";
+import { countries } from "models/countries";
+import { statuses } from "models/statuses";
+
+import CustomersData from "views/customersData";
+
+export default class Data extends JetView {
+	config() {
+		const tabbar = {
+			view: "tabbar",
+			multiview: true,
```

Comment:
get rid of multiview magic: it works with "global" webix ids. 
instead, define local ids for multiview cells and handle tabbar onChange to show them

Reply:
Attached onChange event and configured it's work.